### DISCUSSION TITLE
Flag to know the client already download the PWA?

### DISCUSSION THREAD

**Discussion by jihad-gdt** [khmyznikov/pwa-install#84] [Q&A] [answered]
is there any flag or identifier to know is the user/client already download the PWA? I want to hide the button/instruction if user already add the PWA

**Comment by khmyznikov** [comment, marked as answer]
If **isInstallAvailable** is false, that's means you are already in standalone pwa or in unsupported environment. 
If **isUnderStandaloneMode** is true, that's means you are in standalone pwa mode specifically.
If **isRelatedAppsInstalled** is true, that means PWA or Store App already installed (don't work on iOS)

For iOS, you can only put some variable into Local Storage for example when you see event **pwa-install-how-to-event**, but you can't be sure it was installed.
